draft-ietf-quic-multipath.md

```diff
@@ -1333,6 +1333,8 @@ paths allows.
 An implementation should take care to handle different PMTU sizes across
 multiple paths. As specified in {{Section 14.3 of QUIC-TRANSPORT}} the
 DPLPMTUD Maximum Packet Size (MPS) is maintained for each combination of local and remote IP addresses.
+Note that with the multipath extension multiple paths could use the same 4-tuple
+which do not necessarily need to have the same MPS.
 One simple option, if the PMTUs are similar, is to apply the minimum PMTU of all paths to
```

Comment:
This says the right thing, but does not sound right.

As specified in {{Section 14.3 of QUIC-TRANSPORT}} the
DPLPMTUD Maximum Packet Size (MPS) is maintained for each combination of local and remote IP addresses.
_However, the Internet architecture does not guarantee that the paths chosen for different path IDs will actually
experience the same MPS, even if the paths use the same 4-tuple. It might be more robust to independently
perform DPLPMTUD independently for each path._

Reply:
That's okay for me but my main point was actually that other than with RFC9000, with the multipath extension you can have more than one path per 4-tuple/endpoint pair and therefore the statement to maintain one MSP per IP address pair is less useful.

Reply:
OK. Maybe a simpler suggestion:

_Note that with the multipath extension multiple paths could use the same 4-tuple
but that these paths will not always have the same MPS._

Reply:
My original point was the same as @mirjak 's, but I also like the suggestion of independent DPLPMTUD.
Maybe, to make it a bit shorter, how about:
*However,  due to differences in routing, the paths chosen for different path IDs may not experience the same MPS, even if the paths use the same 4-tuple. It may be more robust to perform DPLPMTUD independently for each path.* ?

Reply:
Or even:
*Note that with the multipath extension multiple paths may use the same 4-tuple
but may have different MPS.* ?